yellowbrick/regressor/residuals.py

```diff
@@ -370,9 +371,15 @@ class ResidualsPlot(RegressionScoreVisualizer):
     line_color : color, default: dark grey
         Defines the color of the zero error line, can be any matplotlib color.
 
-    alpha : float, default: 0.75
-        Specify a transparency where 1 is completely opaque and 0 is completely
-        transparent. This property makes densely clustered points more visible.
+    train_alpha : float, default: 1
```

Comment:
Any particular reason you didn't keep alphas set to default: 0.75

Reply:
No particular reasons for that. It's more of a personal choice. It can be set as per the yellowbrick standards.